server/test/src/main/java/io/deephaven/server/test/FlightMessageRoundTripTest.java

```diff
@@ -681,10 +680,10 @@ public void testGetSchema() {
         final Table tickingTable = ExecutionContext.getContext().getUpdateGraph().sharedLock().computeLocked(
                 () -> TableTools.timeTable(1_000_000).update("I = i"));
 
-        try (final SafeCloseable ignored = LivenessScopeStack.open(scriptSession, false)) {
+        try (final SafeCloseable ignored = LivenessScopeStack.open(scriptSession.getQueryScope(), false)) {
```

Comment:
`scriptSession.getQueryScope` seems slightly silly. Better would be to replace the existing `scriptSession` variable with the `QueryScope` instance. I'd use the member variable over `ExecutionContext.getContext().getQueryScope()`, too.

Reply:
TBH I'm not sure why this has a liveness scope like this anyway - shouldn't we have a liveness scope that is opened in before, closed in after?

Reply:
Agreed given our offline chat. I suspect these are OK without the liveness scopes.